core/src/main/java/com/microsoft/applicationinsights/channel/concrete/inprocess/InProcessTelemetryChannel.java

```diff
@@ -61,11 +61,17 @@
  */
 public final class InProcessTelemetryChannel implements TelemetryChannel {
     private final static int DEFAULT_NUMBER_OF_TELEMETRIES_PER_CONTAINER = 500;
+    private final static int MIN_NUMBER_OF_TELEMETRIES_PER_CONTAINER = 1;
+    private final static int MAX_NUMBER_OF_TELEMETRIES_PER_CONTAINER = 500;
 
     private final static int TRANSMIT_BUFFER_DEFAULT_TIMEOUT_IN_SECONDS = 10;
+    private final static int MIN_TRANSMIT_BUFFER_TIMEOUT_IN_SECONDS = 1;
+    private final static int MAX_TRANSMIT_BUFFER_TIMEOUT_IN_SECONDS = 30;
 
     private final static String DEVELOPER_MODE = "DeveloperMode";
-    private final static String EndpointAddress = "EndpointAddress";
+    private final static String ENDPOINT_ADDRESS = "EndpointAddress";
+    private final static String MAX_TELEMETRY_ITEMS_IN_QUEUE = "MaxTelemetryItemsInQueue";
```

Comment:
It is not just being aligned with Harel but we should strive to be as aligned with the .Net as possible, so we should do a double check here.
Anyway - this shouldn't delay your submission - we can update it later.